### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by vbarshai**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/vbarshai/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
```
Comment (vbarshai) [inline ReadMe.md:1]: This is great but why did you do it that way?

Comment (vbarshai) [inline ReadMe.md:1]: !fix why is this broken?

Comment (vbarshai) [inline ReadMe.md:1]: !fixed ok I fixed it!

**Pull request merged**